modules/virt-creating-vm-wizard-web.adoc

```diff
@@ -5,32 +5,50 @@
 [id="virt-creating-vm-wizard-web_{context}"]
 = Running the virtual machine wizard to create a virtual machine
 
-The web console features a wizard that guides you through *General*, *Networking*, *Storage*, *Advanced*, and *Review* steps to simplify the process of creating virtual machines. All required fields are marked by a ++*++. When the required fields are completed, you can review and create your virtual machine.
+The web console features a wizard that guides you to select a virtual machine template and create a virtual machine. Red Hat virtual machine templates are preconfigured with an operating system image, default settings for the operating system, flavor (CPU and memory), and workload type (server). When templates are configured with a boot source, they are labeled with a customized label text or the default label text *Available boot source*. These templates are ready to be used for creating virtual machines.
+
+When you run the *Create virtual machine from template* wizard, select a template from a list of preconfigured templates, review the settings, and create the virtual machine. If you choose to customize your virtual machine, the wizard guides you through the *General*, *Networking*, *Storage*, *Advanced*, and *Review* steps. All required fields displayed by the wizard are marked by a ++*++.
 
 Network interface cards (NICs) and storage disks can be created and attached to virtual machines after they have been created.
 
-.*Bootable Disk*
+.Procedure
 
-If either `URL` or `Container` are selected as the *Source* in the *General* step, a `rootdisk` disk is created and attached to the virtual machine as the *Bootable Disk*. You can modify the `rootdisk` but you cannot remove it.
+. Click *Workloads* -> *Virtualization* from the side menu.
 
-A *Bootable Disk* is not required for virtual machines provisioned from a *PXE* source if there are no disks attached to the virtual machine. If one or more disks are attached to the virtual machine, you must select one as the *Bootable Disk*.
+. From the *Virtual Machines* tab or the *Templates* tab, click *Create* and select *Virtual Machine with Wizard*.
 
-.Prerequisites
+. In the *Select a template* step, select a template that is configured with a boot source.
 
-* When you create your virtual machine using the wizard, your virtual machine's storage medium must support Read-Write-Many (RWX) PVCs.
+. Click *Next* to go to the *Review and create* step.
```

Comment:
It is an actual step where the user reviews the settings and clicks "Create virtual machine"

Reply:
@lmandavi I meant to ask if it was a "step" in the same way as "General", "Networking", etc. Is *Review and create* an actual UI element? Would rewording it as "Click *Next* to review your settings and create the virtual machine" make sense?

Reply:
HI @sjhala-ccs - Review is a UI element - it is listed in the left pane in a numbered list: 1) General 2) Networking 3) Storage 4) Advanced 5) Review.

Review and create, on the other hand, is the final step where the user creates the VM and exits the wizard.

In the wizard (speaking from memory because I don't have access to a server today), the title for the final step where the user clicks Create Virtual Machine button is:

Review and Create. 

So Review and Create is the final step after General, Networking, Storage etc  because it shows up as the final step.